api/migrations/20220609175823-createRoles.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        return queryInterface.createTable('Role', {
```

Comment:
The table should be called `Roles` to match the pluralized naming convention of all other tables